tools/cleaners.py

```diff
@@ -29,3 +30,15 @@ def parse_keywords_files(file_path):
         keywords_list = []
     finally:
         return keywords_list
+
+
+def get_file_hash(file_path):
+    """Return the md5 hash of a file."""
+    BLOCKSIZE = 65536
+    hasher = hashlib.md5()
+    with open(file_path, 'rb') as f:
+        buf = f.read(BLOCKSIZE)
+        while len(buf) > 0:
+            hasher.update(buf)
+            buf = f.read(BLOCKSIZE)
```

Comment:
This code is fine, but the repetition of the `buf = f.read(BLOCKSIZE)` is a little annoying. Here's one possible alternative:

```python
with open(file_path, 'rb') as f:
    while True:
        buf = f.read(65536)
        if not buf:
            break
        hasher.update(buf)
```

(This is theoretically vulnerable to an infinite loop if somebody does shenanigans that write to the file while you're reading it, but that feels unlikely in this case.)